.github/workflows/cd-pydgraph.yml

```diff
@@ -31,13 +30,20 @@ jobs:
           python -m pip install twine
       - name: Verify that protobufs are checked in
         run: |
-          # should be a no-op
+          # we modify one line in proto/api_pb2_grpc.py as described in readme
```

Comment:
more info in readme: https://github.com/dgraph-io/pydgraph/blob/9c0efb561545baea986ed64654fa8ab6546096ff/README.md?plain=1#L526-L531

Reply:
This is okay! ... but let's spend a little time on this today